pkg/sql/serial.go

```diff
@@ -141,6 +141,7 @@ func (p *planner) generateSerialInColumnDef(
 	if err != nil {
 		return nil, nil, nil, nil, err
 	}
+	seqTyp := defType
```

Comment:
It is confusing that there are 2 similarly named variables `seqTyp` and `seqType`. Could you rename one or both to make it clear what a "type" is in the context of a sequence?

Reply:
Good point, and I renamed it to asIntType, since it maps to the AS <TYPENAME> clause on the create sequence.